packages/block-library/src/image/approvalDialog.tsx

```diff
@@ -0,0 +1,138 @@
+/**
```

Comment:
I see that you're using camelCase for file names, we tend to use kebab-case instead.